app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container py-5 w-50 min-vh-100 text-center d-flex flex-column justify-content-center align-items-center">
+
+  <h2 class="display-1 fw-bold mb-5">Log in</h2>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="field">
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "E-mail adresse", class: "mb-4 rounded-3 py-2 px-5" %>
+    </div>
+
+    <div class="field">
+      <%= f.password_field :password, autocomplete: "current-password", placeholder: "password", class: "mb-4 rounded-3 py-2 px-5" %>
+    </div>
+
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
+      </div>
+    <% end %>
+
+    <div class="d-flex justify-content-center">
+      <%= f.submit "Log in", class: "button is-success is-focused mr-2" %>
+      <%= render "devise/shared/links" %>
+    </div>
```

Comment:
- [ ] Also, the design is not looking similar for the login page. I would recommend using the design, color, font similar to the project requirements.


GIven design | Your project design
:---:|:---:
![Screenshot from 2022-01-14 02-56-40](https://user-images.githubusercontent.com/22434660/149411696-79e59460-d376-479a-a660-47de70c13c60.png) | ![Screenshot from 2022-01-14 02-18-55](https://user-images.githubusercontent.com/22434660/149411567-16b18734-1979-4649-abf4-48e0ba61b651.png)

Reply:
✅ Thank you for your feedback